test/tools/test_tools_target.py

```diff
@@ -16,6 +60,27 @@ def test_channel():
     assert channel.last_who is None
 
 
+def test_channel_eq():
+    name = Identifier('#chan')
+    channel = target.Channel(name)
+
+    assert channel != name
+    assert channel == target.Channel(name)
+    assert channel != target.Channel(Identifier('#not_chan'))
+
+
+def test_channel_sortable():
+    blasphemy = Identifier('#blasphemy')
+    channel = target.Channel(Identifier('#chan'))
+    support = target.Channel(Identifier('#support'))
+    love = target.Channel(Identifier('#love'))
+
+    assert sorted([love, support, channel]) == [channel, love, support]
+
+    with pytest.raises(TypeError):
+        sorted([love, blasphemy, support, channel])
```

Comment:
This makes more sense to me as a direct test of comparison

```suggestion
def test_channel_comparison():
    channel = target.Channel(Identifier('#chan'))
    support = target.Channel(Identifier('#support'))
    love = target.Channel(Identifier('#love'))

    assert channel < love < support

    blasphemy = Identifier('#blasphemy')
    with pytest.raises(TypeError):
        blasphemy < channel
```